modules/siddhi-launcher/pom.xml

```diff
@@ -0,0 +1,379 @@
+        <!--Mapper Extensions-->
+        <dependency>
+            <groupId>org.wso2.extension.siddhi.map.wso2event</groupId>
+            <artifactId>siddhi-map-wso2event</artifactId>
+            <version>${siddhi.map.wso2event.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.extension.siddhi.map.xml</groupId>
+            <artifactId>siddhi-map-xml</artifactId>
+            <version>${siddhi.map.xml.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.extension.siddhi.map.json</groupId>
+            <artifactId>siddhi-map-json</artifactId>
+            <version>${siddhi.map.json.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.extension.siddhi.map.text</groupId>
+            <artifactId>siddhi-map-text</artifactId>
+            <version>${siddhi.map.text.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.extension.siddhi.map.binary</groupId>
+            <artifactId>siddhi-map-binary</artifactId>
+            <version>${siddhi.map.binary.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.extension.siddhi.map.keyvalue</groupId>
+            <artifactId>siddhi-map-keyvalue</artifactId>
+            <version>${siddhi.map.keyvalue.version}</version>
+        </dependency>
+        <!--Store Extensions-->
+        <dependency>
+            <groupId>org.wso2.extension.siddhi.store.solr</groupId>
+            <artifactId>siddhi-store-solr</artifactId>
+            <version>${siddhi.store.solr.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.extension.siddhi.store.mongodb</groupId>
+            <artifactId>siddhi-store-mongodb</artifactId>
+            <version>${siddhi.store.mongodb.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.extension.siddhi.store.rdbms</groupId>
+            <artifactId>siddhi-store-rdbms</artifactId>
+            <version>${siddhi.store.rdbms.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.extension.siddhi.store.hbase</groupId>
+            <artifactId>siddhi-store-hbase</artifactId>
+            <version>${siddhi.store.hbase.version}</version>
+        </dependency>
+        <!--TODO remove after fixing jms-->
+        <dependency>
+            <groupId>org.apache.geronimo.specs</groupId>
+            <artifactId>geronimo-jms_1.1_spec</artifactId>
+            <version>1.1.1</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.extension.siddhi.io.mqtt</groupId>
+            <artifactId>siddhi-io-mqtt</artifactId>
+            <version>${siddhi.io.mqtt.version}</version>
+        </dependency>
+    </dependencies>
+    <build>
+        <extensions>
+        </extensions>
+        <plugins>
+            <plugin>
```

Comment:
define plugins in plugin management and refer from there

Reply:
Resolved in commit  c50edac